src/carousel/components/iframe-authoring.tsx

```diff
@@ -113,7 +142,10 @@ export const IframeAuthoring: React.FC<FieldProps> = props => {
             <div className={css.navButtonField}>
               <label htmlFor={navImageUrl}>Custom Navigation Button Image URL</label>
               <input className="form-control" type="text" id={navImageUrl} defaultValue={navImageUrl} onChange={handleNavImageUrlChange} />
-              <p className="help-block">To customize the button for this slide, specify an image URL. Optimal image size: 150x100 pixels.</p>
+              <p className="help-block">To customize the button for this slide, enter an image URL. Optimal image size: 150x100 pixels.</p>
+              <label htmlFor={navImageAltText}>Custom Navigation Button Image Alt Text</label>
+              <input className="form-control" type="text" id={navImageAltText} defaultValue={navImageAltText} onChange={handleNavImageAltTextChange} />
```

Comment:
I think you might want to replace `{` with `"` in `htmlFor={navImageAltText}` and `id={navImageAltText}`. 
So, it's `htmlFor="navImageAltText"` and `id="navImageAltText"`.
Right now, this ID will change dynamically when the user is typing.

Reply:
Ah, right. Thanks for pointing that out. It's fixed in the latest commit.